spec/requests/organisations_controller_spec.rb

```diff
@@ -1025,11 +1121,11 @@
             end
 
             it "updates the table caption" do
-              expect(page).to have_content("1 organisation found matching ‘#{search_param}’")
+              expect(page).to have_content("1 organisations found matching ‘#{search_param}’")
```

Comment:
i think this change from single to plural does not look correct, might we worth to confirm with a content designer

Reply:
So it's more of a quick change that was easy to do - at the moment it's showing singular for all numbers (we just happen to be testing for 1) - I think it's better to show plural for all numbers until we add proper pluralisation

Reply:
the users page already shows plural for all numbers I think, so it's not even consistent as it stands